src/views/user/UserRequestForm.svelte

```diff
@@ -0,0 +1,241 @@
+<script>
+    let files = [];
+    let deadline = '';
+    let textInput = '';
+  
+    const config = process.env.config;
+    const uploadUrl = config.UPLOAD_URL.toString();
```

Comment:
@Florida-Korir 
Remove the unnecessary `.toString()`. Let it just be:
```suggestion
    const uploadUrl = config.UPLOAD_URL;
```